content/integration/gnome-terminal.md

```diff
@@ -1,12 +1,7 @@
 ---
-title: "Using gnome-terminal with rvm"
+title: "Using gnome-terminal with RVM"
 ---
 
-Other terminals: [LXTerminal](/integration/lxterminal/),
-[XTerm](/integration/xterm/), [Konsole](/integration/konsole/),
-[Zsh + iTerm](/integration/zsh/#zsh-iterm) and
-[anything else](/integration/).
```

Comment:
Ah, you mean people linking from other sites... Well, I am planning to rebuild the docs anyway ;)